pkg/restclient/glustershd.go

```diff
@@ -37,3 +37,39 @@ func (c *Client) SelfHeal(volname string, healType string) error {
 
 	return c.post(url, nil, http.StatusOK, nil)
 }
+
+// SelfHealSplitBrain sends request to start split-brain operations on a volume
+func (c *Client) SelfHealSplitBrain(volname string, operation string, args map[string]string) error {
+	var url string
+	switch operation {
+	case "bigger-file":
+		if _, ok := args["file"]; ok {
```

Comment:
as ` bigger-file` and `latest-mtime` having the same code
I think we can avoid code duplication in the case by using `fallthorugh`